lib/plugins/aws/deploy/lib/cleanupS3Bucket.js

```diff
@@ -2,59 +2,35 @@
 
 const BbPromise = require('bluebird');
 const _ = require('lodash');
+const findAndGroupStacks = require('../utils/findAndGroupStacks');
+const s3ObjectsFromStacks = require('../utils/s3ObjectsFromStacks');
 
 module.exports = {
   getObjectsToRemove() {
     // 4 old ones + the one which will be uploaded after the cleanup = 5
-    const directoriesToKeepCount = 4;
-    const serviceStage = `${this.serverless.service.service}/${this.options.stage}`;
+    const stacksToKeepCount = 4;
+    const service = this.serverless.service.service;
+    const stage = this.options.stage;
```

Comment:
Not sure if we can solely rely on the `this.options` or if we should get the stage somwhere from the `service` object because the stage can be also set via the `provider` property in the `serverless.yml` file.

@eahefnawy what do you suggest here?